### ISSUE TITLE
Couldn’t set app executable

### ISSUE THREAD

**Issue by Delamcode** [Al4ise/Azule#131]
When trying to inject some youtube tweaks I get the following errors (Image because I can’t copy text. Sorry) 
![19E061DB-4419-4659-B1FD-30E954576828](https://user-images.githubusercontent.com/68170410/208986402-4a29adbb-d3c2-4688-b242-6ad3f181f6cc.jpeg)

**Comment by Delamcode** [comment]
I have had this error on Ubuntu Server and Desktop (ARM64). I have all dependencies installed (according to the wiki).

**Comment by Al4ise** [comment]
azule may need to be updated to work on arm64 linux. I'll look into it but for now at least I have no idea how to do it

**Comment by Delamcode** [comment]
Maybe it’s because of ubuntu 22.04 (LTS)?

**Comment by Al4ise** [comment]
No, this shouldnt be a problem, its just that the tools azule uses don't officially run on arm64 linux, only x86

**Comment by Al4ise** [comment]
could you try running the latest build? you should probably do it with the -R flag the first time
